httpx/_models.py

```diff
@@ -1491,6 +1491,20 @@ def __bool__(self) -> bool:
             return True
         return False
 
+    def __repr__(self) -> str:
+        template = "<Cookie {name}={value} for {domain} />"
+
+        cookies_repr = ", ".join(
+            [
+                template.format(
+                    name=cookie.name, value=cookie.value, domain=cookie.domain
+                )
+                for cookie in self.jar
+            ]
+        )
```

Comment:
(Nit) Possible simplification:

```suggestion
        template = "<Cookie {cookie.name}={cookie.value} for {cookie.domain} />"

        cookies_repr = ", ".join(
            [template.format(cookie=cookie) for cookie in self.jar]
        )
```